hknweb/templates/about/abouthkn.html

```diff
@@ -0,0 +1,90 @@
+<!-- {% extends "base.html" %} -->
+{% load static %}
+
+{% block header %}
+<link rel="stylesheet" href="{% static 'css/style.css' %}">
+{% endblock %}
+
+{% block title %}About HKN{% endblock %}
+
+
+{% block heading %}
+About HKN
+{% endblock %}
+
+{% block content %}
+
+<div class='centered'>
+<body>
+  <div class='row'>
+
+    <div class='column left'>
+        <p class='what-about-subtitle'>Eta Kappa Nu (HKN) is the <a href='https://hkn.ieee.org/' target='_blank'>national Electrical and Computer Engineering honor society</a>. Founded in 1915, the Mu chapter at UC Berkeley is among the most active engineering societies at Berkeley. We strive to serve the Electrical Engineering and Computer Science (EECS) community at Berkeley by providing weekly tutoring hours, exam review sessions, department tours, course and staff evaluations, among many other services. Our two offices are located in 290 Cory and 345 Soda near the <a href='https://berkeley.edu/map' target='_blank'>northeastern corner of the UC Berkeley campus</a>.</p>
```

Comment:
Wrapping long lines like this (and the other long lines in this file) would be nice as it would be easier to view changes and edit this file (and that shouldn't affect how the text looks when the page is rendered I believe). More of a nice-to-have though.